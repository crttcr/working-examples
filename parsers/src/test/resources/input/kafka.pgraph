// Example server resources
//
//
## vertex.auto.id.generator = STRING_SEQUENCE
## vertex.auto.id.prefix    = xiv.kafka

(b1: BROKER { name = "broker_1"})
(b2: BROKER { name = "broker_2"})
(t1: TOPIC  { name = "source",      partitions = 16})
(t2: TOPIC  { name = "failure",     partitions =  8})
(t3: TOPIC  { name = "quarantine",  partitions =  8})
(t4: TOPIC  { name = "destination", partitions = 16})
