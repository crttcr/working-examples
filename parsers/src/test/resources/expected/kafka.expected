## vertex.auto.id.generator = STRING_SEQUENCE
## vertex.auto.id.prefix = xiv.kafka
VERTEX: b1
	LABEL: BROKER
		PROP name = "broker_1"
VERTEX: b2
	LABEL: BROKER
		PROP name = "broker_2"
VERTEX: t1
	LABEL: TOPIC
		PROP name = "source"
		PROP partitions = 16
VERTEX: t2
	LABEL: TOPIC
		PROP name = "failure"
		PROP partitions = 8
VERTEX: t3
	LABEL: TOPIC
		PROP name = "quarantine"
		PROP partitions = 8
VERTEX: t4
	LABEL: TOPIC
		PROP name = "destination"
		PROP partitions = 16